eng/common/pipelines/templates/jobs/prepare-pipelines.yml

```diff
@@ -35,166 +35,181 @@ jobs:
     - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
```

Comment:
As part of this change syncing you will need to grant permission to the new service connection to each of these. You can do that by opening the service request and adding them one at a time there.

Reply:
They're all added there.